orienteer-core/src/main/java/org/orienteer/core/component/table/DeleteRowCommandColumn.java

```diff
@@ -68,4 +69,9 @@ protected void onConfigure() {
     protected IModel<String> newLabelModel() {
         return new Model<String>("");
     }
+
+    @Override
+    public Component getFilter(String componentId, FilterForm<?> form) {
```

Comment:
Recommend include this method definition into AbstractMetaColumn and remove from bunch of child classes when it's really not used and returning null.